XenAdmin/Dialogs/OptionsPages/ConnectionOptionsPage.cs

```diff
@@ -107,6 +115,8 @@ private void build()
                 ProxyUsernameTextBox.Visible = false;
                 ProxyPasswordLabel.Visible = false;
                 ProxyPasswordTextBox.Visible = false;
+                AuthenticationMethodPanel.Visible = false;

```

Comment:
build() is called only once, in the ConnectionOptionsPage constructor. The authentication controls are created every time the options dialog is opened, are visible by default, and are made invisible if proxy authentication is disabled; so the visibility works correctly when reopening the dialog after enabling/disabling the registry key. Does the visibility need to update while the options dialog is still open?